### PR TITLE
Update to dask gateway 0.9.0

### PR DESCRIPTION
**Pull request by brl0**

### CODE REVIEW

**Comment by brl0** [comment]
Updated this PR to set the affinity of the dask-gateway components.

File: modules/kubernetes/services/dask-gateway/outputs.tf [hunk 1]
```diff
@@ -7,10 +7,8 @@ output "config" {
   description = "Dask gateway /etc/dask/dask-gateway.yaml configuration"
   value = {
     gateway = {
-      address          = "http://web-public-${var.name}"
-      "proxy-address"  = "scheduler-public-${var.name}:8786"
```
Comment (costrouc) [inline modules/kubernetes/services/dask-gateway/outputs.tf:11]: @brl0 is there a reason that `proxy-address` was not included in new changes?

Comment (brl0) [reply modules/kubernetes/services/dask-gateway/outputs.tf:11]: @costrouc, The scheduler proxy no longer runs as a separate pod and is handled by the traefik proxy now, and the `proxy-address` is inferred from `address`.

**Review by aktech — APPROVED** [review]

**Comment by aktech** [comment]
@brl0 It seems I get the following issue with the traefik
```
 kubectl logs terraform-traefik-traefik-576c9bf66b-b2skg -n dev
```

```
time="2021-03-02T19:34:38Z" level=error msg="Error configuring TLS: secret dev/qhub-cert does not exist" providerName=kubernetes ingress=dask-gateway namespace=dev
time="2021-03-02T19:34:39Z" level=error msg="Error configuring TLS: secret dev/qhub-cert does not exist" providerName=kubernetes namespace=dev ingress=dask-gateway
time="2021-03-02T19:34:39Z" level=error msg="Error configuring TLS: secret dev/qhub-cert does not exist" providerName=kubernetes ingress=dask-gateway namespace=dev
time="2021-03-02T19:34:39Z" level=error msg="Error configuring TLS: secret dev/qhub-cert does not exist" ingress=dask-gateway namespace=dev providerName=kubernetes
```

I haven't got to the root cause yet.

**Comment by aktech** [comment]
Seems like irrelevant to your changes. I have created an issue here: https://github.com/Quansight/qhub-cloud/issues/358

**Pull request merged**